bin/eslint.js

```diff
@@ -56,18 +56,17 @@ function contentBody(check) {
   return content + "Source: http://eslint.org/docs/rules/\n";
 }
 
-function isFileWithMatchingExtension(file, extensions) {
-  var stats = fs.lstatSync(file);
-  var extension = "." + file.split(".").pop();
+function isFileWithMatchingExtension(path, extensions) {
+  var stats = fs.lstatSync(path);
+  var extension = "." + path.split(".").pop();
   return (
     stats.isFile() &&
-    !stats.isSymbolicLink()
-    && extensions.indexOf(extension) >= 0
+    !stats.isSymbolicLink() &&
+    extensions.indexOf(extension) >= 0
   );
 }
 
-function isFileIgnoredByLibrary(file) {
-  var path = file.replace(/^\/code\//, "");
```

Comment:
One of the file strategies has a hard-coded `/code/` prefix which likely relies on that replace for the ignore logic to behave as intended.